core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -59,6 +59,7 @@
     private static final String _SEQ_NO = "_seq_no";
     private static final String RESULT = "result";
     private static final String FORCED_REFRESH = "forced_refresh";
+    private static final String _INPUT = "_input";
```

Comment:
If we're parsing it then, yeah, I'd import it. If we're not parsing it then I wouldn't. To be honest adding the string isn't a big deal either way. At least it is consistent.